packages/devcmd/src/from-cli/index.ts

```diff
@@ -23,6 +25,20 @@ export function run(...args: string[]) {
   });
 }
 
+export async function runReserved(cmd: string) {
+  printDevcmdHeader();
+  process.stdout.write("\n");
+  assertInDevCmdsDir();
+  assertReservedCmdExists(cmd);
+
+  try {
+    const cmdFn = getReservedCommand(cmd);
```

Comment:
to expand on my comment above, is there a reason why we separate that `cmd` is available from actually retrieving the associated `cmdFn`?

`getReservedCommand` could just throw or return null if the command doesn't exist, right?

Reply:
I'll fix this myself and then will manually merge this PR :)